Extensions/Video/JsExtension.js

```diff
@@ -0,0 +1,775 @@
+/**
+ *
+ * JsExtention.js :  Permet de déclarer les events et fait l'affichage des instances d'object dans l'éditeur de GD grace à RenderedVideoObjectInstance
+ *
+ * videoruntimeobject-pixi-renderer.js  : Gère le rendu des object dans la preview et les exports.
+ *
+ * Les actions : a déclarer dans JsExtention.js, puis ça appel dans videoruntimeobject.js, puis vers videoruntimeobject-pixi-renderer.js
+ * Si il y a un operator il faut un .setManipulatedType('number')  et     .setGetter('getVolume');
+ * la nouvelle valeur calculé avec l'opérateur est envoyé directement à setFunctionName("setVolume") !
+ *
+ *
+ */
+
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+  createExtension: function(_, gd) {
+    const extension = new gd.PlatformExtension();
+    extension
+      .setExtensionInformation(
+        "Video",
+        "Video",
+        "Display a video in your scene",
+        "Aurélien vivet",
+        "Open source (MIT License)"
+      )
+      .setExtensionHelpPath("/all-features/video");
+
+    // Declare an object.
+    // Create a new gd.ObjectJsImplementation object and implement the methods
+    // that are called to get and set the properties of the object, as well
+    // as the properties of the initial instances of this object
+    // Everything that is stored inside the object is in "content" and is automatically
+    // saved/loaded to JSON.
+    var videoObject = new gd.ObjectJsImplementation();
+    videoObject.updateProperty = function(
+      objectContent,
+      propertyName,
+      newValue
+    ) {
+      if (propertyName === "Video object opacity") {
+        objectContent.property1 = newValue;
+        return true;
+      }
+      if (propertyName === "Video Looped or no") {
+        objectContent.property2 = newValue === "1";
+        return true;
+      }
+      if (propertyName === "Volume") {
+        objectContent.property3 = newValue;
+        return true;
+      }
+
+      return false;
+    };
+    videoObject.getProperties = function(objectContent) {
+      var objectProperties = new gd.MapStringPropertyDescriptor();
+
+      objectProperties.set(
+        "Video object opacity",
+        new gd.PropertyDescriptor(objectContent.property1)
```

Comment:
opacity will be a number, so you need to cast it to a string:
```
new gd.PropertyDescriptor(objectContent.opacity.toString()).setType(
          "number"
        )
```

Reply:
I've had error with **toString()**

![image](https://user-images.githubusercontent.com/1670670/53280747-2d4bce00-371e-11e9-8d9a-fdbaac45d56c.png)